src/securejoin/bob.rs

```diff
@@ -220,6 +223,33 @@ pub(super) async fn handle_auth_required(
         }
 
         info!(context, "Fingerprint verified.",);
+        if let Some((msg_id, param)) = context
+            .sql
+            .query_row_optional(
+                "
+SELECT id, param FROM msgs
+WHERE chat_id=? AND timestamp=(SELECT MAX(timestamp) FROM msgs WHERE chat_id=?)
```

Comment:
I'm wondering whether we need to add `AND hidden=0` to both WHERE clauses. Might be fine because there is not really any reason why there would be a hidden message inbetween, but also, it wouldn't hurt.

i.e.

```suggestion
WHERE chat_id=? AND hidden=0 AND timestamp=(SELECT MAX(timestamp) FROM msgs WHERE chat_id=? AND hidden=0)
```